hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1313,28 +1420,31 @@ for the Abeta site blocked a direct action of Abeta on nAChRs.
 It is therefore possible that the toxicity of Abeta is mediated,
 at least in part, through a direct physical interaction
 between Abeta and nAChRs."
-
+SET Confidence = "Low"
 SET CellLine="PC12"
 
 a(CHEBI:"amyloid-beta") -> p(HGNC:MAPT, pmod(Ph))
 act(p(HGNCGENEFAMILY:"Cholinergic receptors nicotinic subunits")) -| p(HGNC:MAPT, pmod(Ph))
 act(a(MESH:Bungarotoxins)) -| p(HGNC:MAPT, pmod(Ph))
-
+complex(a(CHEBI:"amyloid-beta"), p(HGNCGENEFAMILY:"Cholinergic receptors nicotinic subunits")) -> path(HBP:neurotoxicity)
```

Comment:
the evidence gives only a prediction and no real results, so i would not add this statement